eth/downloader/downloader.go

```diff
@@ -196,7 +196,11 @@ func New(stateDb ethdb.Database, mux *event.TypeMux, hasHeader headerCheckFn, ha
 // Progress retrieves the synchronisation boundaries, specifically the origin
 // block where synchronisation started at (may have failed/suspended); the block
 // or header sync is currently at; and the latest known block which the sync targets.
-func (d *Downloader) Progress() (uint64, uint64, uint64) {
+func (d *Downloader) Progress() (uint64, uint64, uint64, uint64, uint64) {
```

Comment:
Could you update the docs with a brief description about the new returned state values?